### PR TITLE
docs(core): adding link in README

### PR DESCRIPTION
**Pull request by javier**
adding link to quickstart repository

### CODE REVIEW

**Comment by questdb-butler** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Fails</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:no_entry_sign:</td>
      <td>

  Please update the PR title to match this format:
`type(subType): description`

Where `type` is one of:
`feat`, `fix`, `chore`, `docs`, `style`, `refactor`, `perf`, `test`, `ci`, `chore`, `revert`

And: `subType` is one of:
`build`, `sql`, `log`, `mig`, `core`, `ilp`, `pgwire`, `http`, `conf`, `ui`, `wal`

For Example:

```
perf(sql): improve pattern matching performance for SELECT sub-queries
```
  </td>
    </tr>
  </tbody>
</table>




<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against c6138034fc98b0218a624dc6af2bd122539dab6b
</p>

File: README.md [hunk 1]
```diff
@@ -15,24 +15,23 @@
...
-QuestDB implements ANSI SQL with native time-series SQL extensions. These SQL
-extensions make it simple to correlate data from multiple sources using
-relational and time-series joins. We achieve high performance by adopting a
-column-oriented storage model, parallelized vector execution, SIMD instructions,
-and low-latency techniques. The entire codebase is built from the ground up in
-Java and C++, with no dependencies and zero garbage collection.
+QuestDB implements ANSI SQL with native time-series SQL extensions. These SQL extensions make it simple to correlate
+data from multiple sources using relational and time-series joins. We achieve high performance by adopting a
+column-oriented storage model, parallelized vector execution, SIMD instructions, and low-latency techniques. The entire
+codebase is built from the ground up in Java and C++, with no dependencies and zero garbage collection.
```
Comment (goodroot) [inline README.md:34]:
```suggestion
QuestDB implements ANSI SQL with native time-series SQL extensions. These SQL extensions make it simple to correlate
data from multiple sources using relational and time-series joins. We achieve high performance by adopting a
column-oriented storage model, parallelized vector execution, SIMD instructions, and low-latency techniques. The entire
codebase is built from the ground up in Java and C++, with no dependencies and zero garbage collection.
```

This feedback is likely out of scope for this PR. However, the section is interesting information, but maybe a lower section about "QuestDB internals" would fit better. The reason is that we deliver the core benefit, then state who it is for. After that, we want people to get into "Try QuestDB" as soon as possible.

**Review by goodroot — COMMENTED** [review]
@javier As a noob, the quickstart is much appreciated. 👍 

Some feedback to consider.

File: README.md (hunk 1 above)
Comment (javier) [reply README.md:34]: In this PR I was just adding a link, but for sure if you feel changing the README intro makes sense go for it :). I believe this part in particular was driven mostly by Nic after discussing on Slab about the intro

File: README.md [hunk 2]
```diff
@@ -108,11 +101,16 @@ languages:
 - [Python](https://py-questdb-client.readthedocs.io/en/latest/)
 - [Rust](https://docs.rs/crate/questdb-rs/latest)
 
+### End-to-end quickstart
+
+If you want to go over a quickstart covering batch and streaming ingestion, and data visualisation with QuestDB as the
+database and Grafana for the dashboard, please refer to
+our [quickstart repository](https://github.com/questdb/questdb-quickstart).
```
[the remark this answers is not in the document]
Comment (javier) [reply README.md:108]: I am committing the change, but not sure about changing the structure. I think this should be discussed on the content or general channel, as putting this at the top would mean making the benchmarks or just the getting started using docker and the web console go lower

**Pull request merged**